### PR TITLE
Update README (information for latest version)

### PR DESCRIPTION
**Pull request by gaborh-da**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -146,4 +139,10 @@ Example to fetch at most two SettledDvPs and check if one of them has a payment
 If the boolean argument (`exact`) is true, each incoming contract must match at least one predicate.
 Otherwise, it may skip several events until all predicates are matched.
 
+### Running tests
+
+The library uses JUnit4 to provide its functionalities. Several build systems (e.g. Maven, Sbt) support running JUnit tests easily.
+In Maven, one can use either *maven-failsafe-plugin* or *maven-surefire-plugin* to run tests using this test library.
+To configure any of these, use the official documentation, to run, execute `mvn verify`.
+
 © 2019 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
```
Comment (chrisnorris-da) [inline README.md:148]: shoud it now be 2020?

Comment (gaborh-da) [reply README.md:148]: Honestly, I am not sure which year we should have there. But I don't remember updating it so maybe the year we started the project (2019) is okay.

Comment (darko-da) [reply README.md:148]: I think Copyright needs to be manually updated on a yearly basis. So yeah, all of the copyright notices need to be bumped up.

File: README.md [hunk 2]
```diff
@@ -146,4 +139,10 @@ Example to fetch at most two SettledDvPs and check if one of them has a payment
 If the boolean argument (`exact`) is true, each incoming contract must match at least one predicate.
 Otherwise, it may skip several events until all predicates are matched.
 
+### Running tests
+
+The library uses JUnit4 to provide its functionalities. Several build systems (e.g. Maven, Sbt) support running JUnit tests easily.
+In Maven, one can use either *maven-failsafe-plugin* or *maven-surefire-plugin* to run tests using this test library.
+To configure any of these, use the official documentation, to run, execute `mvn verify`.
+
```
Comment (richardkapolnai-da) [inline README.md:147]: How about something like this?
```suggestion
Example applications using this plugin via Maven: [Bond issuance ref app](https://github.com/digital-asset/ex-bond-issuance), [Healthcare ref app](https://github.com/digital-asset/ex-healthcare-claims-processing), [Supply chain ref app](https://github.com/digital-asset/ex-bond-issuance).

```

**Comment by gaborh-da** [comment]
> looks good to me. Are there other places this should be applied?

Other projects maybe.

Could you also reply to Richard's question about the examples?

File: README.md (hunk 2 above)
Comment (chrisnorris-da) [reply README.md:147]:
```suggestion
Example open source DAML reference applications using this plugin via Maven: [Bond issuance](https://github.com/digital-asset/ex-bond-issuance), [Healthcare](https://github.com/digital-asset/ex-healthcare-claims-processing), [Supply Chain](https://github.com/digital-asset/ex-supply-chain).
```

**Pull request merged**